### ISSUE TITLE
Utf-8 in Schema view

### ISSUE THREAD

**Issue by spiner2000** [ytsaurus/ytsaurus-ui#534]
Cosmetic issue: keys of container types defined with Utf-8 symbols does not displayed properly in schema view of table:

![image](https://github.com/ytsaurus/ytsaurus-ui/assets/1374700/2077a531-5895-4ba5-a872-2ec56e86b4bf)

**Comment by spiner2000** [comment]
same in the tooltip of  row headers

**Comment by ma-efremoff** [comment]
Please describe detailed steps to reproduce

**Comment by spiner2000** [comment]
field example
```json
{
    "name": "СвПрод",
    "required": false,
    "type": "any",
    "type_v3": {
        "type_name": "optional",
        "item": {
            "type_name": "list",
            "item": {
                "type_name": "struct",
                "members": [
                    {
                        "type": "utf8",
                        "name": "КодПрод"
                    },
                    {
                        "type": "utf8",
                        "name": "НаимПрод"
                    },
                    {
                        "type": "utf8",
                        "name": "ПрОтнПрод"
                    }
                ]
            }
        }
    }
}
```
How it looks in schema tab of table:

![image](https://github.com/ytsaurus/ytsaurus-ui/assets/1374700/e2b278fe-f607-42df-94e6-aefc1d02d743)

**Comment by spiner2000** [comment]
Broken encoding of utf-8 field names is in sort menu also.

**Comment by vrozaev** [comment]
Hello @spiner2000, it's should be fixed together with https://github.com/ytsaurus/ytsaurus-ui/issues/533. Could you check that it's works?

**Comment by spiner2000** [comment]
> Hello @spiner2000, it's should be fixed together with #533. Could you check that it's works?
Sorry, I missed this. No, the latest UI still has it.

![image](https://github.com/user-attachments/assets/fa432fad-590a-47f1-a90b-9bb3e632b4bc)
